### DISCUSSION TITLE
How do I clear an entire WASM notebook?

### DISCUSSION THREAD

**Discussion by Andrew-S-Rosen** [marimo-team/marimo#1848] [Q&A] [answered]
Is there a way to delete all the cells in a WASM marimo notebook? I have been somewhat painfully deleting every single cell manually, which is definitely not the most efficient way of achieving this goal...

**Comment by akshayka** [comment, marked as answer]
@Andrew-S-Rosen are you using the playground at `marimo.app`? If so, you can go to `https://marimo.new` to create a blank notebook (overwriting any existing notebook you might have open).

**Reply by Andrew-S-Rosen** [reply]
Ah, perfect! Thanks! That does it.
